index.bs

```diff
@@ -10361,19 +10363,12 @@ is the [=interface=]:
         [=overload resolution algorithm=].
     1.  Let |R| be the result of performing the actions listed in the description of
         |constructor| with |values| as the argument values.
+    1.  Assert: |R| is an object that implements |I|.
```

Comment:
Good point, yeah. Instead maybe let's add "Rethrow any exceptions." to the previous step.